junction/templates/proposals/detail/base.html

```diff
@@ -79,6 +79,10 @@ <h1 class="proposal-title">
                 </div>
             {% endif %}
 
+            {% if proposal.is_first_time_speaker %}
+            <span class="label label-primary">First Time Speaker</span>
```

Comment:
Shouldn't this be inside something like `{% if is_reviewer %}` ?

Reply:
> Shouldn't this be inside something like `{% if is_reviewer %}` ?

Fixed in a new commit.